spec/datadog/core/environment/execution_spec.rb

```diff
@@ -191,6 +191,7 @@ def test_it_does_something_useful
                 Open3.capture3('ruby', stdin_data: script)
               end
 
+              skip('DEBUG')
```

Comment:
After talking with Tony, it seems that Ruby 3.4 outputs an exception here instead of the information to be asserted because of the forked process. I've skipped this test for now, but open to any other fixes!

Reply:
We can mark this test `pending` conditionally with ruby version.

Reply:
Sounds good!